app/main.py

```diff
@@ -1 +1,46 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self):
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        print(f"Eating {self.appetite} food points...")
+        was_hungry = self.is_hungry
+        self.is_hungry = False
+        if was_hungry:
+            return self.appetite
+        return 0
+
+
+class Cat(Animal):
+    def __init__(self, name: str, is_hungry: bool = True):
+        appetite = 3
+        super().__init__(name, appetite, is_hungry)
+
+    def catch_mouse(self) -> None:
+        print("The hunt began!")
```

Comment:
This is     @staticmethod, so rewrite with him

Reply:
You make a good point